### ISSUE TITLE
2.6.12 patch release plan

### ISSUE THREAD

**Issue by joeparsons** [az-digital/az_quickstart#2893]
- #2741
- #2775
- #2719
- #2756 
- #2839
- #2891

## Pick list
```
git cherry-pick db3a59be17ce091715b9c948d643c175a62a5a79
git cherry-pick 4f23fcdd4419a42f4f1972bc3686621ee64dad71
git cherry-pick da389a7a91ae0f0cd50c19d387de3195edf8c606
git cherry-pick 0c27e516f300556a29fa11d47721066be87b2045
git cherry-pick 35e924e575a36794f35d43deb534cafb255c82dd
git cherry-pick 25931f6f3385bd98b88b56952e682d00f9385548
```

**Comment by joeparsons** [comment]
## Release notes snippet
`git log 2.6.11..HEAD --pretty='format:- %s'`
```
_Note: Drupal 9 [reached end of life on November 1, 2013](https://www.drupal.org/psa-2023-11-01).  This will likely be the final 2.6.x release.  Sites still using 2.6.x should upgrade to 2.7.x or 2.8.x ASAP._

_Note: This release contains a security update for the Paragraphs admin module addressing a moderately critical security vulnerability ([SA-CONTRIB-2023-049](https://www.drupal.org/SA-CONTRIB-2023-049))._

- Closes #2890: Update paragraphs_admin to 8.x-1.5 for SA-CONTRIB-2023-049. (#2891)
- Closes #2838 Fix failing create-release workflow. (#2839)
- Closes #2755 Update create release workflow ahead of API changes. (#2756)
- Closes #2718: Update pagination sr-only heading levels (#2719)
- Closes #2774 Undefined array key warning with Text on Media paragraphs (#2775)
- Update drupal/cas requirement from 2.1.0 to 2.2.0 (#2741)
- Prepare 2.6.x branch for new minor release branch 2.8.x (#2811)
- Back to dev.
```

**Comment by joeparsons** [comment]
Release created: https://github.com/az-digital/az_quickstart/releases/tag/2.6.12
